<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>951</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Indicate which antibiotics to use in the resistance profile (profile antibiotics).</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>881</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>You may also include additional antibiotics which will appear in the resistance profile listing (supplementary antibiotics).</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>491</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>Local antibiotic list</string>
   </property>
   <widget class="QListWidget" name="local_antibiotics_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>471</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>471</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Profile antibiotics</string>
   </property>
   <widget class="QListWidget" name="profile_antibiotics_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>451</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_profile_antibiotics">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear List</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>380</y>
     <width>471</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Supplementary antibiotics</string>
   </property>
   <widget class="QListWidget" name="supplementary_antibiotics_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>451</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_list_supplementary">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear List</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="add_profile_antibiotics_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_profile_antibiotics_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_supplementary_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>490</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_supplementary_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>540</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>680</y>
     <width>168</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
